tests/components/deconz/test_sensor.py

```diff
@@ -292,6 +300,8 @@ async def test_add_battery_later(hass):
     assert len(gateway.deconz_ids) == 0
     assert len(gateway.events) == 1
     assert len(remote._callbacks) == 2
+    assert len(gateway.entities[sensor.DOMAIN]) == 0
+    assert len(gateway.entities[EVENT]) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)